libcontainer/process_linux.go

```diff
@@ -151,10 +151,6 @@ func (p *setnsProcess) start() (retErr error) {
 			if werr != nil {
 				logrus.WithError(werr).Warn()
 			}
-			err := ignoreTerminateErrors(p.terminate())
```

Comment:
p.terminate() here is the same as p.ops.terminate() after this PR?

Reply:
Yes, you are right.
`p.terminate()` equals `initProcess.terminate()` or `setnsProcess.terminate()`.

Reply:
Thanks!